<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportManually</class>
 <widget class="QWidget" name="ImportManually">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>汉仪文黑-85W</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(240,240,240);</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>3</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*按钮普通态*/
QPushButton
{
	/*字体为微软雅黑*/
    font-family:汉仪文黑-85W;
    /*字体大小为20点*/
    font-size:25pt;
    /*字体颜色为黑色*/    
    color:black;
    /*背景颜色*/  
    background-color:transparent;
    /*边框圆角半径为25像素*/ 
    border-radius:31px;
	border-image: url(:/Icon/close.png)
}

/*按钮停留态*/
/*QPushButton:hover*/
/*{*/
    /*背景颜色*/  
    /*background-color: #ecf5ff;*/
	/*字体颜色*/  
    /*color: #409eff;*/
/*}*/

/*按钮按下态*/
QPushButton:pressed
{
    /*字体颜色为黑色*/    
    color:black;
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
	border-image: url(:/Icon/close_selected.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="set" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>180</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255 , 255 , 255);
border-radius:10px;</string>
   </property>
   <widget class="QLabel" name="setText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请选择圣遗物套装</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="setBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240,240,240);
border-image: url(:/BG/background.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="position" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>180</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪文黑-85W</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255 , 255 , 255);
border-radius:10px;</string>
   </property>
   <widget class="QLabel" name="positionText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent; </string>
    </property>
    <property name="text">
     <string>请选择部位</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="positionBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);</string>
    </property>
   </widget>
   <widget class="QLabel" name="bgPic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>180</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>bgPic</zorder>
   <zorder>positionText</zorder>
   <zorder>positionBox</zorder>
  </widget>
  <widget class="QWidget" name="levelStar" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>180</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪文黑-85W</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255 , 255 , 255);
border-radius:10px;</string>
   </property>
   <widget class="QLabel" name="starText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent; </string>
    </property>
    <property name="text">
     <string>请输入星级（5）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="starEdit">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>45</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;目前仅支持5星圣遗物&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="levelText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent; </string>
    </property>
    <property name="text">
     <string>请输入等级（0~20）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="levelEdit">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>105</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="mainAttribute" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>120</y>
     <width>180</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255 , 255 , 255);
border-radius:10px;</string>
   </property>
   <widget class="QLabel" name="maText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请选择主属性</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="maBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);</string>
    </property>
   </widget>
   <widget class="QLabel" name="mavText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请输入数值
（百分数请转换小数）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="mavEdit">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>3</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;导入&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">/*按钮普通态*/
QPushButton
{
	/*字体为微软雅黑*/
    font-family:汉仪文黑-85W;
    /*字体大小为20点*/
    font-size:15pt;
    /*字体颜色为黑色*/    
    color:black;
    /*背景颜色*/  
    background-color:transparent;
    /*边框圆角半径为25像素*/ 
    border-radius:21px;
	border-image: url(:/Icon/import.png)
}

/*按钮停留态*/
/*QPushButton:hover
{
    /*背景颜色*/  
    /*background-color: #ecf5ff;
	/*字体颜色*/  
    /*color: #409eff;
}

/*按钮按下态*/
QPushButton:pressed
{
    /*字体颜色为黑色*/    
    color:black;
    /*背景颜色*/  
    background-color:transparent;
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
	border-image: url(:/Icon/import_selected.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="subAttribute1" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>180</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255 , 255 , 255);
border-radius:10px;</string>
   </property>
   <widget class="QLabel" name="sa1Text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请选择副属性①</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="sa1Box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);</string>
    </property>
   </widget>
   <widget class="QLabel" name="sav1Text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请输入数值
（百分数请转换小数）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="sav1Edit">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="subAttribute2" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>180</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255 , 255 , 255);
border-radius:10px;</string>
   </property>
   <widget class="QLabel" name="sa2Text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请选择副属性②</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="sa2Box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);</string>
    </property>
   </widget>
   <widget class="QLabel" name="sav2Text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请输入数值
（百分数请转换小数）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="sav2Edit">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="subAttribute3" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>180</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255 , 255 , 255);
border-radius:10px;</string>
   </property>
   <widget class="QLabel" name="sa3Text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请选择副属性③</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="sa3Box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);</string>
    </property>
   </widget>
   <widget class="QLabel" name="sav3Text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请输入数值
（百分数请转换小数）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="sav3Edit">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="subAttribute4" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>330</y>
     <width>180</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255 , 255 , 255);
border-radius:10px;</string>
   </property>
   <widget class="QLabel" name="sa4Text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请选择副属性④</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="sa4Box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);</string>
    </property>
   </widget>
   <widget class="QLabel" name="sav4Text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请输入数值
（百分数请转换小数）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="sav4Edit">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪文黑-85W</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240,240,240);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>9</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪文黑-85W</family>
     <pointsize>10</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(211, 188, 142);</string>
   </property>
   <property name="text">
    <string>导入圣遗物（手动）</string>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>back</zorder>
  <zorder>set</zorder>
  <zorder>position</zorder>
  <zorder>levelStar</zorder>
  <zorder>mainAttribute</zorder>
  <zorder>save</zorder>
  <zorder>subAttribute1</zorder>
  <zorder>subAttribute2</zorder>
  <zorder>subAttribute3</zorder>
  <zorder>subAttribute4</zorder>
  <zorder>text</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
